### PR TITLE
Updates after 6 month review of Swedish page 

### PR DESCRIPTION
**Pull request by LianeHughes**
Relatively minor additions and corrections to align with recent changes

### CODE REVIEW

**Review by hedi-ee — COMMENTED** [review]
The text and structure of the page look good. I just stumbled upon one link that might be broken for good (https://experience.arcgis.com/experience/09f821667ce64bf7be6f9f87457ed9aa) and a domain (http://pathogens.se/) that might be only temporarily out.

File: national-resources/sweden.md [hunk 1]
```diff
@@ -1,121 +1,146 @@
...
+- [COVID symptom study Sweden](https://www.pathogens.se/dashboards/symptom_study_sweden/): a compilation of resources related to the COVID symptom study Sweden project, which monitored the occurrence of COVID-19 symptoms across different regions of Sweden to estimate the prevalence and distribution of symptomatic cases (no longer updated as of July 2022).
+- [COVID-19 test statistics from the National Pandemic Centre](https://www.pathogens.se/dashboards/npc-statistics/): shows the number of tests conducted each week, and the relative proportions of positive, negative, and inconclusive results over time (no longer updated as of December 2020).
+- [The Swedish Public Health Agency’s COVID-19 statistics](https://experience.arcgis.com/experience/09f821667ce64bf7be6f9f87457ed9aa): shows data on the number of cases, admissions to intensive care, and mortality related to COVID-19 over time (only available in Swedish).
```
[the remark this answers is not in the document]
Comment (LianeHughes) [reply national-resources/sweden.md]: removed the experience.arcgis link and set pathogens link to a DNS currently working (temporary issues)

File: national-resources/sweden.md [hunk 2]
```diff
@@ -1,121 +1,146 @@
...
-- [The National Board for Health and  Welfare’s dashboard on COVID-19 statistics](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/): shows information on the number of COVID-19 cases, and survival rates, as well as information on post COVID-19 condition/longcovid both for different regions and over time (only available in Swedish).
-- [The National Board for Health and  Welfare’s dashboard on data indirectly related to COVID-19](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/statistik-relaterad-till-covid-19/): shows information related to the number of influenza cases over time, as well as general mortality rates in Sweden (only available in Swedish and no longer updated as of January 2022).
+- [SARS-CoV-2 antibody tests conducted by the SciLifeLab Autoimmunity and Serology profiling facility](https://www.pathogens.se/dashboards/serology-statistics/): shows the total number of tests conducted, as well as the number of positive and negative tests over time.
```
[the remark this answers is not in the document]
Comment (LianeHughes) [reply national-resources/sweden.md]: We have having temporary issues, but this link should now work regardless (will continue to work after too)

**Comment by LianeHughes** [comment]
Realised that I had accidentally pushed some work on the data description page here too. Apologies for this. I have left some of the changes in that I think make no difference (e.g. adding the tool and new collaborators)

**Pull request merged**